### PR TITLE
account-summary: show account sync progress m/n synced

### PR DESCRIPTION
**Pull request by Tomasvrba**

### CODE REVIEW

File: frontends/web/src/routes/account/summary/initializeall.tsx [hunk 1]
```diff
@@ -91,13 +91,15 @@ class InitializeAllAccounts extends Component<Props, State> {
...
-    public render({ t }: RenderableProps<Props>) {
+    public render({ t }: RenderableProps<Props>, { initialized }: State) {
         if (this.allInitialized()) {
             return <AccountsSummary />;
         }
         return (
             <div>
-                <Spinner text={t('accountSummary.synchronizing')}/>
+                <Spinner text={t('accountSummary.synchronizing') +
+                Object.values(initialized).filter(status => status === true).length +
```
Comment (KasparEtter) [inline frontends/web/src/routes/account/summary/initializeall.tsx]: `filter(status => status === true)` -> `filter(status => status)`?

File: frontends/web/src/routes/account/summary/initializeall.tsx [hunk 2]
```diff
@@ -91,13 +91,15 @@ class InitializeAllAccounts extends Component<Props, State> {
...
+                `/${this.props.accounts.length}`}/>
```
Comment (KasparEtter) [inline frontends/web/src/routes/account/summary/initializeall.tsx]: Rather use `render({ t, accounts }: RenderableProps<Props>, …` to be consistent.

File: frontends/web/src/locales/en/app.json [hunk 3]
```diff
@@ -24,7 +24,7 @@
     "fiatBalance": "Fiat Balance",
     "name": "Account Name",
     "noAccount": "There are no accounts to show.",
-    "synchronizing": "Synchronizing all accounts...",
+    "synchronizing": "Synchronizing accounts with the blockchain: ",
```
Comment (KasparEtter) [inline frontends/web/src/locales/en/app.json]: I forgot whether we are allowed to change values here. Only if they have not yet been translated? And maybe "with their [respective] blockchain"? Or shall we sacrifice accuracy for easier understanding?

Comment (Tomasvrba) [reply frontends/web/src/locales/en/app.json]: I agree that "with their respective blockchains" would be nice to be technically correct, but I think we should go for simplicity in this case. Also easier to translate.

Comment (Tomasvrba) [reply frontends/web/src/locales/en/app.json]: If we get a lot of hate for it from grammar nazis then we can change it :fearful:

Comment (Tomasvrba) [reply frontends/web/src/locales/en/app.json]: Changed back here and made the change in locize

Comment (KasparEtter) [reply frontends/web/src/locales/en/app.json]: Great, thanks!

**Pull request merged**